test_conformance/math_brute_force/binary.c

```diff
@@ -1536,23 +1646,691 @@ static cl_int TestDouble( cl_uint job_id, cl_uint thread_id, void *data )
 
 }
 
-int TestFunc_Float_Float_Float(const Func *f, MTdata d)
-{
-    return TestFunc_Float_Float_Float_common(f, d, 0);
-}
+// A table of more difficult cases to get right
+static const cl_half specialValuesHalf[] = {
+   0xffff,0x0000,0x0001
```

Comment:
could have some more values in here: +/- INF, -0, HALF_MAX, smallest & largest normal